pipelines/ApPipeWithFakes.yaml

```diff
@@ -109,8 +109,8 @@ contracts:
       transformDiaSrcCatWithFakes.connections.ConnectionsClass(config=transformDiaSrcCatWithFakes).diaSourceCat.name
   - imageDifferenceWithFakes.connections.ConnectionsClass(config=imageDifferenceWithFakes).subtractedExposure.name ==
       diaPipe.connections.ConnectionsClass(config=diaPipe).diffIm.name
-  - imageDifferenceWithFakes.connections.ConnectionsClass(config=imageDifferenceWithFakes).warpedExposure.name ==
-      diaPipe.connections.ConnectionsClass(config=diaPipe).warpedExposure.name
+  - imageDifferenceWithFakes.connections.ConnectionsClass(config=imageDifferenceWithFakes).template.name ==
+      diaPipe.connections.ConnectionsClass(config=diaPipe).template.name
```

Comment:
When I was running with this ticket branch for DM-33911 I was getting an error on BPS submission due to this line. If I instead did

```
imageDifference.connections.ConnectionsClass(config=imageDifference).inputTemplate.name ==
         diaPipe.connections.ConnectionsClass(config=diaPipe).template.name
```
it works.

Reply:
Oops, I meant to put that comment in ApPipe.yaml, but this one will presumably have the same problem (change to config=imageDifferenceWithFakes of course).